### ISSUE TITLE
[feature] Add alt text to images

### ISSUE THREAD

**Issue by ianlewis** [ianlewis/www.ianlewis.org#339]
"Alt-text" should be added to all images. The content of each image should be examined and an accessibility description added as alt text.

The markdownlint `MD045/no-alt-text` check should be enabled and should pass. The `htmlvalidate` check `wcag/h30` should also be enabled and should pass.

**Comment by ianlewis** [comment]
HTML validation now passes because everything that's left is empty alt text (e.g. `alt=""`) which `html-validate` treats as purely decorative.

The remaining changes that are needed can be retrieved from `todos`.

```shell
 ➜  todos --label \#339
```
